spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -233,20 +233,23 @@ module.exports = function (provider, setup) {
   //
   // Note that this also implicitly tests the close() method of client sockets.
   it("onConnection sockets dispatch onDisconnect", function(done) {
+    var cspy = jasmine.createSpy('client'),
+        client,
+        receiver;
     var onDispatch = function (evt, msg) {
       expect(evt).toEqual('onDisconnect');
-      done();
+      socket.close(done);
```

Comment:
OK, looks like this is still testing the same thing -- I guess I didn't try this in Chrome, now I see that the other tests have this `socket.close(done);` pattern too. Thanks for fixing!